transport.go

```diff
@@ -118,3 +134,15 @@ func (wb *wrappedBody) Close() error {
 	wb.span.End()
 	return wb.body.Close()
 }
+
+func getenv(key string, fallback int) int {
```

Comment:
Take a look at config.go , we have the configs in `Config`, we should probably keep everything there, also the parameters for limiting the size.